modules/virt-snocaps-differences.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.doc
+
+:_content-type: CONCEPT
+[id="virt-snocaps-differences_{context}"]
+= Single Node {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster. See xref:../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] to prepare your cluster for installation.
+
+When provisioning an {product-title} {sno} cluster with the assisted installer, out-of-the-box persistent storage is auto-deployed using the {product-title} link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/deploying_openshift_data_foundation_on_single_node_openshift_clusters/index#doc-wrapper[Data Foundation Logical Volume Manager Operator].
+
+* In {VirtProductName} 4.10 and 4.11, the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Hostpath Provisioner (HPP)] is automatically installed.
+* In {VirtProductName} 4.12, the link:https://cloud.redhat.com/blog/using-the-openshift-assisted-installer-service-to-deploy-an-openshift-cluster-on-metal-and-vsphere[assisted installer] for {sno} automatically launches when a {sno} configuration is detected and guides you through the installation. Alternatively, you can also deploy using the HPP.
```

Comment:
So as I mentioned above I think we should have LVMO here, right now we are not stressing the fact that it is the provided storage on 4.12.

What happens when you provision a 4.12 SNO cluster with CNV,
is that assisted installer will additionally install LVMO, so I would consider something like:
```
In OpenShift Virtualization 4.12, the OpenShift Container Platform Data Foundation Logical Volume Manager Operator (link) is the provided out-of-the-box storage solution.
Alternatively, it is still possible to set up HPP for use manually.
```

Reply:
Done.